### ISSUE TITLE
Firebird 5.0.1.1469 Access violation. The code attempted to access a virtual address without privilege to do so.

### ISSUE THREAD

**Issue by OlegMatveevFB** [FirebirdSQL/firebird#8337]
SRV-DB-041	Tue Dec  3 02:27:04 2024
	 Access violation.
		The code attempted to access a virtual
		address without privilege to do so.
	This exception will cause the Firebird server
	to terminate abnormally.

Firebird 5.0.1.1469, ODS 13.1, Dialect 1
5 peices [core dumps](https://drive.google.com/drive/folders/1M5y8dYugCmI9HnCg081_n6Af0Gzjtdan?usp=sharing)

**Comment by hvlad** [comment]
It is related with huge amount of monitoring data. Could you confirm ?

**Comment by OlegMatveevFB** [comment]
Yes, sure. The suspicion was initially about a connection trigger making a request to mon$attachments.
The problem happened twice.
The first time was on one database, after which I turned off this trigger. I enabled the creation of dumps and left the trigger enabled in another database.
But in FB 2.5.9 it works.

**Comment by OlegMatveevFB** [comment]
Added SQL file into 
[bag folder](https://drive.google.com/drive/folders/1M5y8dYugCmI9HnCg081_n6Af0Gzjtdan?usp=sharing)

**Comment by hvlad** [comment]
It is definitely bug. And, perhaps, already fixed in 5581ea6 
If it is reproducible, could you try with latest snapshot build ?

**Comment by OlegMatveevFB** [comment]
Last snapshot can try only after December 24. The error is not specifically reproduced. On November 24th I migrated from 2.5.9 to 5.0.1. This is only the second time.

**Comment by hvlad** [comment]
Ok, lets wait for @dyemanov opinion

**Comment by hvlad** [comment]
Any news ?

**Comment by OlegMatveevFB** [comment]
25.12 .2024 was installed 5.0.2.1583
No problems until today
